RealifeTech-CoreSDK/Storage/Queue/QueueItem.swift

```diff
@@ -8,7 +8,13 @@
 
 import Foundation
 
-struct QueueItem <T: Codable> {
-    let item: T
-    let releaseQueue: (_: QueueAction) -> Void
+public struct QueueItem <T: Codable> {
```

Comment:
Can we rename `T` to `Item`

Reply:
Good one. I've changed all generic type name to more specific names